eo-maven-plugin/src/main/java/org/eolang/maven/ResolveMojo.java

```diff
@@ -202,8 +220,36 @@ private Collection<Dependency> deps() throws IOException {
             .collect(Collectors.toList());
     }
 
+    /**
+     * Check if all dependencies have transitive dependencies.
+     *
+     * @param deps Dependencies
+     * @throws java.lang.IllegalStateException if a transitive dependency is found
+     */
+    private void checkTransitive(final Collection<Dependency> deps) {
+        for (final Dependency dep : deps) {
+            if (this.hasTransitiveDependencies(dep)) {
+                throw new IllegalStateException(
+                    String.format("%s contains transitive dependencies", dep)
+                );
+            }
+        }
+    }
+
+    /**
+     * Check if dependency has transitive dependencies.
+     *
+     * @param dep Dependency
+     * @return True if has transitice dependencies
+     */
+    private boolean hasTransitiveDependencies(final Dependency dep) {
```

Comment:
@volodya-lombrozo Same here. Can we make this name simpler?

Reply:
@Graur Fixed. Please, take a look.